internal/rpcserver/rpcserver.go

```diff
@@ -5582,25 +5627,73 @@ type rpcserverConfig struct {
 	// of to provide additional data when queried.
 	TxIndex   *indexers.TxIndex
 	AddrIndex *indexers.AddrIndex
+
+	// NetInfo defines a slice of the available networks.
+	NetInfo []types.NetworksResult
+
+	// MinRelayTxFee defines the minimum transaction fee in Atoms/1000 bytes to be
+	// considered a non-zero fee.
+	MinRelayTxFee dcrutil.Amount
+
+	// Proxy defines the proxy that is being used for connections.
+	Proxy string
+
+	// These fields define the username and password for RPC connections and
+	// limited RPC connections.
+	RPCUser      string
+	RPCPass      string
+	RPCLimitUser string
+	RPCLimitPass string
```

Comment:
Not related to this pr, but would it be prudent to store the `authsha` that these values become rather than keeping the plain-text in memory?

Reply:
From my understanding, this is not much of an issue since accessing the RPC server from a remote/untrusted network is not recommended.  However, as a best practice, I agree that the exposure of plaintext passwords should be minimized (its also stored in plaintext in the config file and/or passed through the command line).

Reply:
Not a bad idea, but realistically, if someone has enough access to do that, it's probably game over in short order anyway.

Reply:
It's been fine for this long, so it's probably not an issue. The rpc password usually is something random anyway, not something personal.